ports/sbcl.lisp

```diff
@@ -87,20 +87,47 @@
 
 ;;;; Utilities for writing VOPs
 
+#+sb-cga-sse2
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  ;; check if we should use sb-vm::ea or make-ea
+  ;; ea was added in 1.4.11
+  ;; make-ea was removed after 2.0.11
+  ;; use make-ea before SBCL 2.0.0
+  (when (< (parse-integer (lisp-implementation-version) :junk-allowed t) 2)
```

Comment:
Would be better to directly check MAKE-EA, along the lines of:

```suggestion
  (when (fboundp (find-symbol "MAKE-EA" :sb-vm))
```

Can you check if this works? (Don't have SBCL handy myself right now.)

Reply:
Works fine. Thanks.
Committed and pushed the change – now I see that one can "Commit suggestion" directly from here…

Reply:
**WAIT A MOMENT** - the earliest I tested with was SBCL 1.4.12 - that already has EA.

Reply:
It works with SBCL 1.4.10 as well.